Mapbox/MapboxMapsFoundation/Camera/CameraManager.swift

```diff
@@ -220,11 +220,13 @@ public class CameraManager {
         let clampedZoom: CGFloat? = zoom?.clamp(withMax: mapCameraOptions.maximumZoomLevel,
                                                 andMin: mapCameraOptions.minimumZoomLevel)
 
+        let optimalBearing = optimizeBearing(startBearing: mapView.bearing, endBearing: bearing)
```

Comment:
What happens if bearing is greater than 360 degrees here?

Reply:
Can you set a bearing greater than 360 degrees?

Reply:
Nothing's stopping a developer from doing it, so yep.